### ISSUE TITLE
Possible upgrade

### ISSUE THREAD

**Issue by HaKaShis** [type76/fluid#1]
Hi, thanks for the work you did,

Could you maybe make a version with all the original settings and for the music to play not on the center but as the SPACE bar effect ?

And maybe its possible react to the mic sound ?

**Comment by type76** [comment]
Hey, thanks for your idea for adding music to it actually.
 
Are you looking at the [v2.html ](https://type76.com/fluid/v2.html) version?
I don't think I quite understand when you say 'the space bar effect'.
Do you mean to play when the user presses space bar?
For mic input, I will have to look into this. It should be possible.

Your initial idea was to project this as a DJ backdrop?

**Comment by HaKaShis** [comment]
Yeah, simulate the beat to the user "presses", or what I mean as "random splats" on PavelDoGreat git.

right now yours simulates the spalts in the center, it goes white and thats it, it would be much better if the whole screen would be random populated by the beat as "random splats" does right now.

**Comment by type76** [comment]
When it goes all white, I think it's a settings thing. A gentle acoustic tune looks ok at these settings, but the when the beat drops with a dance track it blows out. I'll fix this in the next update I hope.

For the splats on space bar press, the default browser behaviour does this no? Click the button, then space bar repeats the click. Is that what you mean?

I'm going in different directions, so I will add some goals in the readme. Your initial idea made me think of a crazy disco dancing device for my finger so thats project 2. The mic input control is best as an on/off button in the HUD right?

**Comment by type76** [comment]
I think I need to put an issue on my own github thing here. This readme will get out of hand if I continue like this.

I'm not a programmer either and no one taught me github. I see these 'Assignees' and 'milestones' on the side there. Although I'd call them roles and goals. You want to press them?

**Comment by HaKaShis** [comment]
I'll try and make a video of what I imagine, maybe it will be easier to explain my idea. :D

**Comment by type76** [comment]
or I can help you code it yourself.
